moviebot/nlu/annotation/rule_based_annotator.py

```diff
@@ -223,9 +228,11 @@ def _title_annotator(
                     and len([int(val) for val in re.findall(r"\b\d+", gram)])
                     == 0
                 ):
-                    # check if
-                    # all words are in the list of stop words and no numbers
+                    # check if no words are in the list of stop words and
+                    # there are no numbers; This is used for partial matching.
```

Comment:
Can we rephrase it a bit? Now it isn't very clear. Maybe something like: _Check if there are no words that are included in the list of stop words or that are numbers._

Reply:
Done